ext/node/polyfills/process.ts

```diff
@@ -934,6 +937,10 @@ internals.__bootstrapNodeProcess = function (
     platform = isWindows ? "win32" : Deno.build.os;
     pid = Deno.pid;
 
+    // this is the llvm version used in rustc 1.79.0
+    Process.prototype.config["variables"]["llvm_version"] =
+      Deno.build.os === "darwin" ? "18.1" : "0.0";
```

Comment:
How do we ensure this version is up-to-date when the rust version changes?

Reply:
Yeah I don't actually even know if that's the right value to be honest. In node it's the llvm version that node gets built with, but I don't really know how that translates to us, or how much it matters.

Reply:
Is there a way to get the llvm version in build.rs? If so, then we could do a check similar to what we do with the typescript version.

Reply:
Apparently we can actually just always set it to `0.0`, so that simplifies this